packages/core/src/environment.ts

```diff
@@ -0,0 +1,61 @@
+export type Network = {
+  id: string
+  description: string
+  chain_id: number // >= 0
+  live: boolean
+  default_provider: string // a valid HTTP url pointing at a RPC endpoint
+  gas?: string // e.g. '1 gwei'
+  gas_multiplier: number // e.g. 1.1
+  native_token_name: string
+  hopr_token_name: string
+  tags: string[]
+}
+
+export type Environment = {
+  id: string
+  network_id: string // must match one of the Network.id
+  channel_contract_deploy_block: number // >= 0
```

Comment:
Sure it could. But that requires chain interaction for information which is static once available. However, we might remove this if we don't find any use for it.

Reply:
It is very beneficial to have this information because we then know since which block we need to start indexing.